scripts/dev_setup.sh

```diff
@@ -151,6 +152,27 @@ function install_helm {
   fi
 }
 
+function install_minikube {
+  if ! command -v minikube &> /dev/null; then
+    if [[ $(uname -s) == "Darwin" ]]; then
+      install_pkg minikube brew
+    else
+      MACHINE=$(uname -m);
+      if [[ $MACHINE == "x86_64" ]]; then
+        MACHINE="amd64"
+      fi
+      TMPFILE=$(mktemp)
+      rm "$TMPFILE"
+      mkdir -p "$TMPFILE"/
+      curl -sL -o "$TMPFILE"/out "https://storage.googleapis.com/minikube/releases/v${MINIKUBE_VERSION}/minikube-$(uname -s | tr '[:upper:]' '[:lower:]')-${MACHINE}"
```

Comment:
this version of installation would work for mac as well if we want, and could pin the version if we think that's needed.   Not sure it's worth changing though.

Reply:
probably not needed. I followed the convention in the other deps we install, e.g. use `brew` when convenient.